### ISSUE TITLE
Questions about MFRL

### ISSUE THREAD

**Issue by Jianxff** [zkyseu/lane2seq.github.io#1]
Hi,
That's a great work to apply sequence generation to lane detection!

I'm implementing the code refering to the paper, and I've done the basic part for anchor format. The pretrining stage works, but I got stuck in the MFRL part.

I'm not familiar with RL, but the main idea of Task-Reward model tuning looks not complicated so I want to have a try. Unfortunately I cannot make it. I have some questions:
- How the `batch_sample` works?  According to the appendix B of the paper: 'draw two outputs from one training image', how are `gradient` and `baseline` sampled ?  
- How to calculate the gradient? The cross-entropy is used during pretraining. But refering to the paper of Task-Reward, the gradient is calculated without GT. So I want to know how the gradient is calculated during model tuning. 

Thanks a lot !
